src/core/glob_matcher.h

```diff
@@ -8,6 +8,16 @@
 #include <string>
 #include <string_view>
 
+#define FIX_PERFORMANCE_MATCHING
+
+#ifndef FIX_PERFORMANCE_MATCHING
```

Comment:
You first define `FIX_PERFORMANCE`  and then you `ifndef` it. Isn't that a bug ?

Reply:
`FIX_PERFORMANCE` is a bad name.

it should be REFLEX_PERFORMANCE